wagtail/admin/api/actions/move.py

```diff
@@ -0,0 +1,40 @@
+from django.core.exceptions import ValidationError as DjangoValidationError
+from django.shortcuts import get_object_or_404
+from rest_framework import fields, status
+from rest_framework.exceptions import ValidationError
+from rest_framework.response import Response
+from rest_framework.serializers import Serializer
+
+from wagtail.core.actions.move_page import MovePageAction
+from wagtail.core.models import Page
+
+from .base import APIAction
+
+
+class MovePageAPIActionSerializer(Serializer):
+    target_page_id = fields.IntegerField(required=True)
```

Comment:
I think for consistency with copy, we should rename this to ``destination_page_id``

Reply:
Alright!